buildscripts/semantic-convention/generate.sh

```diff
@@ -4,7 +4,8 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR="${SCRIPT_DIR}/../../"
 
 # freeze the spec version to make SemanticAttributes generation reproducible
-SPEC_VERSION=v1.1.0
+SPEC_VERSION=a44d863edcdef63b0adce7b47df001933b7a158a # = v1.1.0 + number->int update; bump to v1.2.0 once released
```

Comment:
We could also put this on hold until spec v1.2.0 is released and re-generate the constants again if it includes updates.
```suggestion
SPEC_VERSION=v1.2.0
```

Reply:
yes, we do not want to run against un-released spec versions now that we've hit 1.0. I'll add a tag to this PR that it's blocked on the spec release.

Reply:
There were no semconv changes other than the number->int one (https://github.com/open-telemetry/opentelemetry-specification/compare/v1.1.0...a44d863edcdef63b0adce7b47df001933b7a158a) and thus the *Attributes.java files were not touched when I ran the script. Either is fine with me.

Reply:
I want to draw a pretty hard line of not updating things until the spec is released. We should not try to chase spec updates, and I don't want maintainers to have to verify shas in order to confirm that there aren't any "substantive" changes.